Mlem/App/Views/Shared/EndOfFeedView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  EndOfFeedView.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-08-19.
+//
+
+import Foundation
+import MlemMiddleware
+import SwiftUI
+
+struct EndOfFeedViewContent {
+    let icon: String
+    let message: String
```

Comment:
`message` needs to be a `LocalizedStringResource`, or `EndOfFeedViewContent` needs to have an initialiser that accepts one

```suggestion
    let message: LocalizedStringResource
```